Test/WebTestCase.php

```diff
@@ -79,22 +80,38 @@ protected function tearDown()
     }
 
     /**
+     * @return ContainerInterface
```

Comment:
What about mocking `$container` and force it by setting the property? You can override protected properties visibility using reflection.
Another option could be mocking calls to `$kernel::createKernel` and `$kernel->getContainer()`